light/store/db/db.go

```diff
@@ -289,32 +277,28 @@ func (s *dbs) Size() uint16 {
 	return s.size
 }
 
-func (s *dbs) lbKey(height int64) []byte {
-	return []byte(fmt.Sprintf("lb/%s/%020d", s.prefix, height))
+func (s *dbs) sizeKey() []byte {
+	// heights always starts from 1. Hence in order to avoid
+	// key collisions we reserve size to be the key at height 0.
```

Comment:
This sounds like a bad idea to me. Some poor developer in the future (it might be you!) is going to iterate over the `lbKey` prefix and trip over the size value. It's also not extensible, if we want to add additional metadata later. Use a separate prefix or sub-prefix for metadata and heights.